src/App.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+
+import React, { useEffect } from 'react';
+import { NavigationContainer, useFocusEffect } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { NativeModules } from 'react-native';
+import { RootStackParamList } from './navTypes';
+import HomeScreen from './HomeScreen'
+import ProductScreen from './ProductScreen'
+
+const Stack = createNativeStackNavigator<RootStackParamList>();
+
+const { AttentiveCreativeModule } = NativeModules;
+
+function App(): JSX.Element {
+  useEffect(() => {
```

Comment:
does this mean that we call identify every time the home screen is rendered? should we add a comment saying ideally we call identify every asap & then when we have new identifying data?

Reply:
Nope, the second parameter to `useEffect` is an array of dependencies. Setting that dependency list to `[]` means the effect only runs once.